ple/ple.py

```diff
@@ -1,416 +1,420 @@
-import numpy as np
-from PIL import Image  # pillow
```

Comment:
Same comment as `snake.py`. Can you adjust this so only the relevant changes show up?